chain/x/zoracle/internal/types/msgs_test.go

```diff
@@ -480,3 +480,83 @@ func TestMsgEditOracleScriptGetSignBytes(t *testing.T) {
 
 	require.Equal(t, expected, string(res))
 }
+
+func TestMsgSetOracleAddress(t *testing.T) {
+	validator := sdk.ValAddress([]byte("validator"))
+	reporter := sdk.AccAddress([]byte("reporter"))
+	msg := NewMsgAddOracleAddress(validator, reporter)
+	require.Equal(t, RouterKey, msg.Route())
+	require.Equal(t, "add_oracle_address", msg.Type())
+	require.Equal(t, validator, msg.Validator)
+	require.Equal(t, reporter, msg.Reporter)
+
+	cases := []struct {
```

Comment:
Normally we separate this into different unit tests? Opinion @taobun ?

Reply:
Let's break it